### PR TITLE
chore: fix some comments

### PR DESCRIPTION
**Pull request by tianzedavid**
## Description

 fix some comments

## Related Issue(s)

- Closes #[issue number]

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please describe):

## Breaking Change

If this PR introduces a breaking change, please provide a detailed description of the impact and the migration path for existing applications.

## Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have run `cargo fmt` and `cargo clippy` to ensure my code is formatted and linted correctly
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots (if applicable)

Please include any relevant screenshots or GIFs that demonstrate the changes made.

## Additional Notes

Please provide any additional information or context that may be helpful for reviewers.

### CODE REVIEW

File: pallets/subtensor/src/lib.rs [hunk 1]
```diff
@@ -1552,7 +1552,7 @@ pub mod pallet {
         // 		- The work has been performed on a stale, future, or non existent block.
         //
         // 	* 'InvalidDifficulty':
-        // 		- The work does not match the difficutly.
+        // 		- The work does not match the difficultly.
```
Comment (distributedstatemachine) [inline pallets/subtensor/src/lib.rs:1555]:
```suggestion
        // 		- The work does not match the difficulty.
```

File: pallets/subtensor/src/registration.rs [hunk 2]
```diff
@@ -205,7 +205,7 @@ impl<T: Config> Pallet<T> {
     // 		- The work has been performed on a stale, future, or non existent block.
     //
     // 	* 'InvalidDifficulty':
-    // 		- The work does not match the difficutly.
+    // 		- The work does not match the difficultly.
```
Comment (distributedstatemachine) [inline pallets/subtensor/src/registration.rs:208]:
```suggestion
    // 		- The work does not match the difficulty.
```

File: pallets/subtensor/tests/difficulty.rs [hunk 3]
```diff
@@ -27,7 +27,7 @@ fn test_registration_difficulty_adjustment() {
         SubtensorModule::set_max_registrations_per_block(netuid, 3);
         SubtensorModule::set_max_allowed_uids(netuid, 3);
         SubtensorModule::set_network_registration_allowed(netuid, true);
-        assert_eq!(SubtensorModule::get_difficulty_as_u64(netuid), 20000); // Check set difficutly.
+        assert_eq!(SubtensorModule::get_difficulty_as_u64(netuid), 20000); // Check set difficultly.
```
Comment (distributedstatemachine) [inline pallets/subtensor/tests/difficulty.rs:30]:
```suggestion
        assert_eq!(SubtensorModule::get_difficulty_as_u64(netuid), 20000); // Check set difficulty.
```

**Review by distributedstatemachine — CHANGES_REQUESTED** [review]
Thanks so much for contributing !! A couple of nits and we should be able to merge this

**Comment by darrenkel** [comment]
Is this spam? Perhaps the @tianzedavid is farming trust? Github profile looks suspicious

**Comment by paul-opentensor** [comment]
This account, tianzedavid, has been blocked due to suspicious account activity. OT DM me via internal comms if there are questions or concerns.

**Comment by distributedstatemachine** [comment]
> Is this spam? Perhaps the @tianzedavid is farming trust? Github profile looks suspicious

**Pull request closed without merging**